victor_fake_hardware_interface/scripts/minimal_fake_control_mode_interface_node.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+
+import argparse
+import sys
+
+import rospy
+from victor_fake_hardware_interface.minimal_fake_arm_interface import MinimalFakeControlModeInterface as FCMI
+from victor_hardware_interface_msgs.msg import ControlMode
+
+
+def main(args):
```

Comment:
Seems fine, but I am curious why you need to run just the control mode interface.

Reply:
this is how I mock this interface for simulation. We need it in order for arm_robots stuff to run, since it makes calls to it, but I didn't happen to need the motion command/status topics, so I didn't want them to run since they would be wrong/confusing.